### ISSUE TITLE
Missing require for `TimeoutError / Timeout::Error`

### ISSUE THREAD

**Issue by ffleming** [vicvega/hpfeeds-ruby#6]
```
🐩  forrestfleming@fsf-work ~ % pry
>> require 'hpfeeds'
=> true
>> ::HPFeeds::Client.new(host:   'localhost', port:   10_000, ident:  'abc', secret: '123', reconnect: false)
W, [2017-04-27T14:45:05.574814 #37703]  WARN -- : NameError caught while connecting: uninitialized constant HPFeeds::TimeoutError. Reconnecting in 20 seconds...
NameError: uninitialized constant HPFeeds::TimeoutError
	from: /Users/forrestfleming/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/hpfeeds-0.1.7/lib/hpfeeds/exception.rb:4:in `<module:HPFeeds>'
	from: /Users/forrestfleming/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/hpfeeds-0.1.7/lib/hpfeeds/exception.rb:1:in `<top (required)>'
	from: /Users/forrestfleming/.rbenv/versions/2.3.3/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from: /Users/forrestfleming/.rbenv/versions/2.3.3/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from: /Users/forrestfleming/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/hpfeeds-0.1.7/lib/hpfeeds/client.rb:62:in `rescue in connect'
	from: /Users/forrestfleming/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/hpfeeds-0.1.7/lib/hpfeeds/client.rb:57:in `connect'
	from: /Users/forrestfleming/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/hpfeeds-0.1.7/lib/hpfeeds/client.rb:42:in `block in tryconnect'
	from: /Users/forrestfleming/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/hpfeeds-0.1.7/lib/hpfeeds/client.rb:40:in `loop'
	from: /Users/forrestfleming/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/hpfeeds-0.1.7/lib/hpfeeds/client.rb:40:in `tryconnect'
	from: /Users/forrestfleming/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/hpfeeds-0.1.7/lib/hpfeeds/client.rb:36:in `initialize'
	from: (pry):2:in `new'
>>
```

PR incoming 😃

**Comment by vicvega** [comment]
It should be fixed in the current release (`0.1.8`). Let me know

Thanks for your contribution!

**Comment by ffleming** [comment]
Yup, confirmed fixed. Thanks!
